src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -0,0 +1,745 @@
+---
+title: "NerdGraph tutorial: Manage your synthetic monitors"
+tags:
+    - NerdGraph
+    - Synthetic
+metaDescription: Use NerdGraph to programatically manage your synthetic monitors, private locations, and secure credentials.
+---
+
+Our NerdGraph API allows you to automate creating, updating, or deleting your synthetic monitors.
+
+If you've never used NerdGraph, read [Intro to NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) for a more thorough tutorial. If you want to use the UI instead, check out [Add and edit monitors](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors/).
+
+## Use queries to retrieve data [#query]
+
+Queries make requests to fetch data. Check out our [NerdGraph entities API tutorial](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial/) to learn additional query capabilities available to your synthetic monitors. 
```

Comment:
when it comes to sentences like this second one, I usually like to start with the thing they're doing. So instead I'd do 'To learn additional...'. To me, this sets them up to be able to quickly know what that is and skip it if it doesn't apply. Starting with 'See our .... for...' makes it harder to parse for me. a small thing and do what you wish with that input, but in case helpful.